docs/wiki/99_Developers/Developers_Guide.md

```diff
@@ -0,0 +1,228 @@
+    `https://github.com/github-username/LORIS`  
+    
+Any development should be done on your fork of LORIS, and **not** on aces/LORIS. 
+Before starting any PR, be mindful of which branch you want to base off of (main vs release-specific branch).   
+
+You will make a different branch for each PR or change that you are making.  
+
+In general, smaller pull requests or changes are easier to work with in our team review context and will flow more smoothly through the review process.  
+
+Here is a basic workflow if you are trying to make a PR off of the _main_ branch:
+
+    git fetch aces
+    git checkout aces/main 
+    git checkout -b "branch_name"
+    vi filename
+    git add filename 
+    git status 
+    git commit -m "Initial commit"
+    git push my_fork branch_name
+
+There is usually no need to fully update your fork to the newest aces/LORIS version, since you can work off whichever branch you just fetched and checked-out.
+
+ **Never** push to aces/Loris. Your branch will be deleted. Always push to your fork, and then create the PR from GitHub.
+
+#### Branch Naming
+Many LORIS developers follow the convention: “date-title...”.   
+Starting with the date allows branches to sort and display in order, and also helps sort out any conflicts faster in the main Loris repository. Please keep branch names short and informative.
+
+#### Development Mode
+To activate the development mode, turn sandbox to 1 in the `project/config.xml` file:
+```
+<dev>
+    <sandbox>1</sandbox>
+</dev>
+```
+And then re-compile:
+```
+npm run compile
+```
+This will change several configurations in your LORIS project to allow for more verbose error reporting, which is useful for debugging.
+
+### Helpful commands to run before pushing new code:
+Travis is our continuous integration which runs on every PR in GitHub. These commands can catch small formatting errors that will cause the Travis build to fail.
+
+    make checkstatic  # includes these 3 commands: 
+                      # npm run lint:php, npm run lint:javascript, vendor/bin/phan
+
+    vendor/bin/phpcs --standard=test/LorisCS.xml modules/candidate_list/php/candidate_list.class.inc
+                      # phpcs allows you to specifically check the formatting of one file 
+
+    vendor/bin/phpcbf --standard=test/LorisCS.xml modules/candidate_list/php/candidate_list.class.inc
+                      # phpcbf allows you to automatically fix the errors 
+                      # found by phpcs marked with a check.
+
+Running unit or integration tests (useful if making significant changes to any module). More details can be found [here](../../../test/README.md):
+         
+    npm run tests:unit
+    # The commands below allow you to run unit tests for a specific 
+    # module or a specific test in a module
+    npm run tests:unit -- --filter CandidateTest
+    npm run tests:unit -- --filter CandidateTest::someTest
+    npm run tests:integration
+
+### Good practice for Travis CI
```

Comment:
Travis CI is fortunately no longer with us. maybe remove references to it and replace with GitHub Actions or automated tests